installer/bundle/bundle_skel.sh

```diff
@@ -1126,18 +1058,22 @@ case "$installMode" in
                 fi
             fi
 
-            # Now actually install of the "queued" packages
-            if [ -n "${add_list}" ]; then
-                if [ "$INSTALLER" = "DPKG" ]; then
-                    [ -z "${forceFlag}" ] && FORCE="--refuse-downgrade" || FORCE=""
-                    dpkg ${DPKG_CONF_QUALS} --install $FORCE ${add_list}
-                else
-                    [ -n "${forceFlag}" ] && FORCE="--force" || FORCE=""
-                    rpm -ivh $FORCE ${add_list}
+            # Install OMS Agent
+            pkg_add ${OMS_PKG} omsagent
+            TEMP_STATUS=$?
+            if [ $TEMP_STATUS -ne 0 ]; then
+               echo "$OMS_PKG package failed to install and exited with status $TEMP_STATUS"
+               cleanup_and_exit $OMS_INSTALL_FAILED
+            fi
+
+            # Install DSC
+            if shouldInstall_omsconfig; then
+                pkg_add ${DSC_PKG} omsconfig
+                TEMP_STATUS=$?
+                if [ $TEMP_STATUS -ne 0 ]; then
+                    echo "$DSC_PKG package failed to install and exited with status $TEMP_STATUS"
+                    cleanup_and_exit $DSC_INSTALL_FAILED
```

Comment:
Looks good except I remember when installing bundle we ignored DSC package failure and will print a message saying that DSC is not installed and these functionalities would not be available. Would that still be the case with this change?

Reply:
Other than my comments above I sign off.

Reply:
Have you tested upgrade scenario with this change?

Reply:
I forgot about the fact that we deliberately don't handle DSC failure and still continue. Do you want me to continue even if DSC fails now too?

Jagadeesh has done 64bit testing for both install and upgrade scenarios, and both worked fine. Could not test 32bit due to Jenkins build break, but for now 64bit testing should be good as changes are not specific to platform.

Reply:
let us keep it consistent with the rest of package installs. I sign off. please merge.